chrome/content/prefs.xul

```diff
@@ -11,50 +11,104 @@
   <prefpane id="otr-pane">
```

Comment:
I merged some of these changes in e7420da0ed8e71a55b7b2f4bc2ca5bb89e87bbd8 so that the coniks diff is clearer.  Please rebase on that as well.